examples/vanilla/index.php

```diff
@@ -9,10 +9,12 @@
 
 function setupSentry()
 {
-    (new \Raven_Client(SENTRY_DSN))
-        ->setAppPath(__DIR__)
+    $sentryClient = new Raven_Client(SENTRY_DSN);
+    $__DIR__ = version_compare(PHP_VERSION, '5.3.0', '>=') ? __DIR__ : dirname(__FILE__);
```

Comment:
maybe, because dirname is defined in all php versions, you can use it always, without checking the PHP_VERSION const